catalog/spatial/geocoding/spatial-geocoding-admin-module/src/test/java/org/codice/ddf/spatial/admin/module/TestGeocodingMbean.java

```diff
@@ -54,6 +54,7 @@ public void setUp() {
     geoEntryIndexer = Mockito.mock(GeoEntryIndexer.class);
     geocoding.setGeoEntryIndexer(geoEntryIndexer);
     geocoding.setGeoEntryExtractor(geoEntryExtractor);
+    System.setProperty("ddf.home", System.getProperty("user.dir"));
```

Comment:
Whats the reasoning behind overriding `ddf.home` to the `user.dir`?

Reply:
Shouldn't we do a temp directory or something along those lines in stead of the user directory?

Reply:
I didn't look into what the tests were testing or how. I only made the directories match what they previously were to my changes.